.devcontainer/devcontainer.json

```
@@ -1,79 +1,43 @@
-      "settings": {
-        "terminal.integrated.profiles.linux": {
-          "fish": {
-            "path": "fish"
-          }
-        },
-        "terminal.integrated.defaultProfile.linux": "fish"
-      },
+	"name": "Docker outside of Docker",
+	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+	"image": "openinf/fisher-ubuntu:latest",
 
-      // Add the IDs of extensions you want installed when the container is created.
-      "extensions": [
-        // https://marketplace.visualstudio.com/items?itemName=aaron-bond.better-comments
-        "aaron-bond.better-comments",
-        // https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint
-        "davidanson.vscode-markdownlint",
-        // https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint
-        "dbaeumer.vscode-eslint",
-        // https://marketplace.visualstudio.com/items?itemName=dprint.dprint
-        "dprint.dprint",
-        // https://marketplace.visualstudio.com/items?itemName=editorconfig.editorconfig
-        "editorconfig.editorconfig",
-        // https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode
-        "esbenp.prettier-vscode",
-        // https://marketplace.visualstudio.com/items?itemName=Gruntfuggly.todo-tree
-        "Gruntfuggly.todo-tree",
-        // https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-docker
-        "ms-azuretools.vscode-docker",
-        // https://marketplace.visualstudio.com/items?itemName=KatjanaKosic.vscode-json5
-        "KatjanaKosic.vscode-json5",
-        // https://marketplace.visualstudio.com/items?itemName=Shopify.theme-check-vscode
-        "Shopify.theme-check-vscode",
-        // https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker
-        "streetsidesoftware.code-spell-checker",
-        // https://marketplace.visualstudio.com/items?itemName=tamasfe.even-better-toml
-        "tamasfe.even-better-toml",
-        // https://marketplace.visualstudio.com/items?itemName=unifiedjs.vscode-remark
-        "unifiedjs.vscode-remark"
-      ]
-    }
-  },
-  // Use 'forwardPorts' to make a list of ports inside the container available
-  // locally.
-  "forwardPorts": [
-    // Jekyll server
-    4000,
-    // Live reload server
-    35729,
-    // OpenSSH server
-    2222
-  ],
+	"features": {
+		"ghcr.io/devcontainers/features/docker-outside-of-docker:1": {
+			"version": "latest",
+			"enableNonRootDocker": "true",
+			"moby": "false"
+		},
+		"ghcr.io/devcontainers/features/node:1": {},
+		"ghcr.io/devcontainers/features/sshd:1": {},
+		"ghcr.io/meaningful-ooo/devcontainer-features/homebrew:2": {},
+		"ghcr.io/devcontainers-contrib/features/curl-homebrew:1": {},
+		"ghcr.io/devcontainers-contrib/features/fish-apt-get:1": {},
+		"ghcr.io/devcontainers-contrib/features/ts-node:1": {},
+		"ghcr.io/devcontainers-contrib/features/turborepo-npm:1": {},
```

```suggestion
		"ghcr.io/devcontainers-contrib/features/turborepo-npm:1": {},
```

removing some of these helps!